README.md

```diff
@@ -15,7 +15,7 @@ connected adaptive quadtrees or octrees in parallel.
 ## Installation
 If you have not yet installed Julia, please [follow the instructions for your
 operating system](https://julialang.org/downloads/platform/). P4est.jl works
-with Julia v1.6 and up.
+with Julia v1.6.
```

Comment:
Isn't this misleading now? Technically, the package works with Julia 1.6 *and* 1.7, only bindings regeneration does not work. Or am I messing something?

Reply:
As you said, not everything works with v1.7. But I see your point, just `using P4est` and doing your thing works absolutely fine on v1.7 which is much more important than being able to generate new bindings I guess. I don't have a strong opinion here, should I revert the commit?

Reply:
Please correct me if I am wrong: If we support everything in 1.7 *except* generating the bindings, I'd say we support "1.6 an up", since *out-of-the-box usage* is supported for 1.7. Only if folks need to regenerate bindings, they need to use 1.6, but this is imho *custom usage*.

Thus if my description of the state of things above is correct, I'd prefer to revert the commit.

Reply:
You're right, I reverted the commit.